### PR TITLE
Pager horizontal alignment

### PR DESCRIPTION
**Pull request by siengnonta**
| | Left  | Center  | Right  | 
|---|---|---|---|
| Default | Simulator Screen Shot - iPhone Xs - 2019-08-27 at 15 54 31 | Simulator Screen Shot - iPhone Xs - 2019-08-27 at 15 54 25 |  Simulator Screen Shot - iPhone Xs - 2019-08-27 at 15 59 24 |
| With Edge Spacing | Simulator Screen Shot - iPhone Xs - 2019-08-27 at 15 54 36 | Simulator Screen Shot - iPhone Xs - 2019-08-27 at 15 54 25 | Simulator Screen Shot - iPhone Xs - 2019-08-27 at 15 59 18 |

### CODE REVIEW

**Review by zntfdr — APPROVED** [review]

File: Sources/FSPageViewLayout.swift [hunk 1]
```diff
@@ -142,8 +144,17 @@ class FSPagerViewLayout: UICollectionViewLayout {
...
-        attributes.center = center
+        var midX: CGFloat = frame.midX
+        if horizontalAlignment == .left {
+            let offset = (collectionViewSize.width - frame.width)/2
+            let edgeSpacing = needsEdgeSpacing ? actualInteritemSpacing : 0
+            midX -= (offset - edgeSpacing)
+        } else if horizontalAlignment == .right {
+            let offset = (collectionViewSize.width - frame.width)/2
+            let edgeSpacing = needsEdgeSpacing ? actualInteritemSpacing : 0
+            midX += (offset - edgeSpacing)
```
Comment (abadikaka) [inline Sources/FSPageViewLayout.swift:155]: both left and right is the same for line 149 - 150
can u make it shorter again ?

**Review by abadikaka — APPROVED** [review]
you can also ask PR to their repo as portfolio @siengnonta

File: Sources/FSPageViewLayout.swift (hunk 1 above)
Comment (siengnonta) [reply Sources/FSPageViewLayout.swift:155]: Yeah, of course.

Comment (siengnonta) [reply Sources/FSPageViewLayout.swift:155]: I try making it shorter but it just reduces one line of code which also reduces readability.
So, leaving the duplicated code in each condition is easier.

**Pull request merged**